src/ForemanInterface.php

```diff
@@ -3,14 +3,13 @@
 
 namespace Neighborhoods\Kojo;
 
+use Neighborhoods\Kojo\Api\V1\RDBMS\Connection;
+use Neighborhoods\Kojo\Api\V1\Worker;
 use Neighborhoods\Kojo\Service\Update;
-use Neighborhoods\Kojo\Api\V1\Worker\ServiceInterface;
 
-interface ForemanInterface
+interface ForemanInterface extends Connection\Service\AwareInterface, Worker\Service\AwareInterface
```

Comment:
I think it's misleading to have this kojospace actor's interface extend `Api/` interfaces. It functionally does, yeah, but semantically they shouldn't be coupled

Reply:
Happy to remove them here, but in LS, my plan is to have all of the workers themselves extend these as necessary. This will then make it easy for the Facade to do an `instance of` check before injecting each service without requiring the workers to always implement everything.

Reply:
```suggestion
interface ForemanInterface
```